html/utilities/validation/setValidSelect.js

```diff
@@ -1,4 +1,4 @@
-function setValidSelect(inputContainer) {
```

Comment:
@mantasio Thanks for the PR! Let's get a semi-colon added to the end of this function and we'll get it merged!

Reply:
sure, done!

Reply:
@mantasio Last thing, could you update this branch with staging?

Reply:
@guern1kn done!